liip/imagine-bundle/1.8/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Liip\\ImagineBundle\\LiipImagineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
+    },
+    "copy-from-package": {
+    	"Resources/config/routing.yaml": "%ETC_DIR%/routing/imagine.yaml"
```

Comment:
My understanding was that this would work, and if it doesn't I believe it should. As for the choice to expose our full routes, this decision was made after a discussion by different contributors and maintainers; we are intentionally exposing the full routing configuration.